pkg/monitor/dynkeepalived.go

```diff
@@ -42,7 +44,13 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 			if err != nil {
 				return err
 			}
-			if prevConfig == nil || prevConfig.VRRPInterface != newConfig.VRRPInterface {
+			curConfig = &newConfig
+			if appliedConfig == nil || !cmp.Equal(*appliedConfig, *curConfig) {
+				if prevConfig == nil || cmp.Equal(*prevConfig, *curConfig) {
+					configChangeCtr++
+				} else {
+					configChangeCtr = 1
+				}
```

Comment:
Shouldn't you need to apply changes only if configChangeCtr > some_threshold?

Reply:
We can address the second part in another pull.